runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/metrics/MetricsTest.java

```diff
@@ -0,0 +1,23 @@
+package io.aklivity.zilla.runtime.engine.metrics;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.test.internal.metrics.TestMetrics;
+
+public class MetricsTest
+{
+    @Test
+    public void shouldResolve()
+    {
+        Configuration config = new Configuration();
+        Metrics metrics = new TestMetrics(config);
+        Metric testCounterMetric = metrics.resolve("test.counter");
+
+        assertThat(testCounterMetric, instanceOf(TestCounterMetric.class));
+
+    }
```

Comment:
Please rename to `shouldResolveCounter`, and add `shouldResoleGauge`, `shouldResolveHistogram`.